### PR TITLE
Add logging in core modules

### PR DESCRIPTION
**Pull request by Ajayneethikannan**
1.Added get_logging_function for use in core modules
2. Add logging for base authentication and session authentication

### CODE REVIEW

File: omniport/core/session_auth/views/logout.py [hunk 1]
```diff
@@ -22,9 +26,11 @@ def get(self, request, *args, **kwargs):
         :return: the response for request
         """
 
+        user = request.user
+        
         # This is a direct replacement for django.contrib.auth.logout()
         SessionMap.delete_session_map(request=request)
-
+        session_auth_log('Successfully logged out', 'info', user)
```
Comment (ayushjn20) [inline omniport/core/session_auth/views/logout.py]: Log message contains no information: Who got successfully logged out and which session?

File: omniport/core/session_auth/views/login.py [hunk 2]
```diff
@@ -64,6 +69,10 @@ def post(self, request, *args, **kwargs):
             response_data = {
                 'errors': serializer.errors,
             }
+            session_auth_log(
+                f'Invalid login attempted with credentials {serializer.data}',
```
Comment (ayushjn20) [inline omniport/core/session_auth/views/login.py]: Log message contains less information: Which `person` attempted an invalid login?

File: omniport/core/session_auth/views/logout.py (hunk 1 above)
Comment (Ajayneethikannan) [reply omniport/core/session_auth/views/logout.py]: Done: added session key, and user info

File: omniport/core/session_auth/views/login.py (hunk 2 above)
Comment (Ajayneethikannan) [reply omniport/core/session_auth/views/login.py]: Done: removed password from being printed on the log

Comment (Ajayneethikannan) [reply omniport/core/session_auth/views/login.py]: Please check the resolved changes

**Comment by dhruvkb** [comment]
Why is everything being logged to `core`? Why not put logs for `session-auth`, `token-auth`, `guest-auth` etc. all separately?

**Comment by Ajayneethikannan** [comment]
@dhruvkb Actually this was what was suggested by Praduman and Ayush when I was working on it :sweat_smile:  . I will change it with separate logs for every module if you want

**Comment by dhruvkb** [comment]
@pradumangoyal @ayushjn20 why did you want all logs to go in one module? Wouldn't debugging be easier if logs were classified app-wise?

**Pull request merged**